bulbs/videos/templates/videos/series-page.html

```diff
@@ -0,0 +1,27 @@
+{% load betty core %}
+
+<header>
+  <h1 id="series-title"></h1>
+  <p>Watch The Latest Episode</p>
+</header>
+
+<figure class="video-container">
+  <div class="content">
+    <bulbs-video
+      twitter-handle="avclub"
+      src="{{ VIDEOHUB_BASE_URL }}/video/"
```

Comment:
I feel like the `latest-episode` js module should actually add the `bulbs-video` element to the page.  Wouldn't this try to first make a request to the base url before latest episode appends the `<video-id>.json` ?  Maybe @collin can weigh in too